### PR TITLE
Top skip

### PR DESCRIPTION
**Pull request by ghislainfourny**

### CODE REVIEW

File: queries/public/api/facttable-for-component.jq [hunk 1]
```diff
@@ -278,6 +282,12 @@ let $serializers := {
         <title type='text'>Cell store facts</title>
         <updated>{current-dateTime()}</updated>
        {
+            let $next := $res."FactTable@odata.nextLink"
+            where exists($next)
+            return <link rel="next">{$next}</link>,
```
Comment (dknochen) [inline queries/public/api/facttable-for-component.jq]: Could be simplified to:

    $res."FactTable@odata.nextLink" ! <link rel="next">{$next}</link>

**Comment by dknochen** [comment]
@ghislainfourny very good start :+1:

One more question: How does it work in the html and spreadsheet view? In the html view we could easily add a first and last row containing a next link in case a nextLink is returned. What do you think?

File: queries/public/api/facttable-for-component.jq (hunk 1 above)
Comment (ghislainfourny) [inline queries/public/api/facttable-for-component.jq]: True, but more difficult to read. Your choice! ;-)

**Comment by ghislainfourny** [comment]
Thanks!

Regarding the spreadsheet, I would leave it aside for now, that's a lot of work.

Regarding html: I agree. I will do this.

Kind regards,
Ghislain

**Comment by ghislainfourny** [comment]
There you go ;-)

http://secxbrl-dev10.28.io/v1/_queries/public/api/entities.jq?format=html&tag=DOW30&token=foobar&top=5

**Comment by dknochen** [comment]
@ghislainfourny I don't see the change in the link you posted. Also, there is a merge conflict.

**Pull request merged**